### PR TITLE
Fix permissions error when loading users MODUIMP-40

### PR DESCRIPTION
**Pull request by adamdickmeiss**
While 100% coverage, the mock does not check for X-Okapi-Token presence.

### CODE REVIEW

**Review by MikhailFokanov — APPROVED** [review]
Looks good. Thanks.

File: src/main/java/org/folio/rest/impl/UserImportAPI.java [hunk 1]
```diff
@@ -45,13 +45,13 @@
 import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 import io.vertx.core.http.HttpMethod;
-import io.vertx.core.impl.CompositeFutureImpl;
+import io.vertx.core.impl.future.CompositeFutureImpl;
```
Comment (julianladisch) [inline src/main/java/org/folio/rest/impl/UserImportAPI.java:48]: Please do not use internal class CompositeFutureImpl, this is not part of the Vert.x API: https://vertx.io/docs/apidocs/
Use https://github.com/folio-org/okapi/blob/master/okapi-common/src/main/java/org/folio/okapi/common/GenericCompositeFuture.java instead.

File: src/main/java/org/folio/service/CustomFieldsService.java [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
-import io.vertx.core.impl.CompositeFutureImpl;
+import io.vertx.core.impl.future.CompositeFutureImpl;
```
Comment (julianladisch) [inline src/main/java/org/folio/service/CustomFieldsService.java:21]: Please do not use internal class CompositeFutureImpl, this is not part of the Vert.x API: https://vertx.io/docs/apidocs/
Use https://github.com/folio-org/okapi/blob/master/okapi-common/src/main/java/org/folio/okapi/common/GenericCompositeFuture.java instead.

File: src/main/java/org/folio/service/DepartmentsService.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
 import java.util.stream.Collectors;
 
 import io.vertx.core.Future;
-import io.vertx.core.impl.CompositeFutureImpl;
+import io.vertx.core.impl.future.CompositeFutureImpl;
```
Comment (julianladisch) [inline src/main/java/org/folio/service/DepartmentsService.java:16]: Please do not use internal class CompositeFutureImpl, this is not part of the Vert.x API: https://vertx.io/docs/apidocs/
Use https://github.com/folio-org/okapi/blob/master/okapi-common/src/main/java/org/folio/okapi/common/GenericCompositeFuture.java instead.

File: src/main/java/org/folio/rest/impl/UserImportAPI.java (hunk 1 above)
Comment (adamdickmeiss) [reply src/main/java/org/folio/rest/impl/UserImportAPI.java:48]: Agreed. I'll make a PR to fix this.

**Pull request merged**